Source/Csla.test/PropertyInfo/PropertyInfoTests.cs

```diff
@@ -69,5 +69,12 @@ public void TestStringNullDefaultValue()
       Assert.AreEqual(null, PropertyInfoRoot.StringNullDefaultValueProperty.DefaultValue);
       Assert.AreEqual(null, PropertyInfoRoot.NewPropertyInfoRoot(dataPortal).StringNullDefaultValue);
     }
+
+    [TestMethod]
+    public void TestContainingType()
```

Comment:
As already written as a comment n the PR. This test does not check the fix from `PropertyInfo.cs`.
When you revert the fix the test is still green.

Reply:
@StefanOssendorf  Thank you for reviewing the PR. You're right, I have just amended the `ContainingTypeProperty` using the `PropertyInfo` constructor changed. I confirm that reverting the change the test now fails. :-)